tests/unitTests/test.cpp

```diff
@@ -0,0 +1,94 @@
+/* Copyright (c) Stanford University, The Regents of the University of California, and others.
+ *
+ * All Rights Reserved.
+ *
+ * See Copyright-SimVascular.txt for additional details.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject
+ * to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+ * IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
+ * PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER
+ * OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include "test.h"
+
+using namespace mat_fun;
+using namespace std;
+
+
+TEST(UnitTest, nHK) {
+    // Step 1: define parameters
+    auto matType = consts::ConstitutiveModelType::stIso_nHook;   // Material_model: options refer to consts.h 
+    auto volType = consts::ConstitutiveModelType::stVol_ST91;   // Dilational_penalty_model
+    double E = 1e6;   // Elasticity_modulus
+    double nu = 0.5;   // Poisson_ratio
+    double pen = 4e9;   // Penalty_parameter
+    double C01;   // additional parameter to C10 (optional)
+
+    // Step 2: construct test object
+    UnitTestIso nHK(matType, E, nu, volType, pen, C01);
+
+    // Step 3: define the input 
+    double F[3][3]; 
+    memset(F, 0, sizeof(F));   // initialize
+    F[0][0] = 1.0; F[1][1] = 1.0; F[2][2] = 1.0;   // set to Identity
+
+    // Step 4: define the reference output 
+    double S_ref[3][3], Dm_ref[6][6];
+    memset(S_ref, 0, sizeof(S_ref));
+    memset(Dm_ref, 0, sizeof(Dm_ref));
+
+    // Step 5: run unit test
+    nHK.runUnitTest(F, S_ref, Dm_ref);
+      
+}
+
+TEST(UnitTest, MR) {
```

Comment:
Do these need different names? I only see this output:
```
Test project /Users/runner/work/svFSIplus/svFSIplus/build/svFSI-build/Source/svFSI
    Start 1: UnitTest
1/1 Test #1: UnitTest .........................   Passed    0.39 sec

100% tests passed, 0 tests failed out of 1

Total Test time (real) =   0.40 sec
```
Shouldn't there be two tests?

Reply:
No, I think the 'Test #1' here means all test cases inside the 'test.cpp' (we can rename it also). It's a little bit confusing, but based on my understanding, no matter how many test cases we put in the 'test.cpp' file, there is only one "test" for `ctest`, because in the CMakeLists: `add_test(NAME all_unit_tests COMMAND run_all_unit_tests)` defines the "test". However, if you type `ctest -VV` or `ctest -verbose`, it will print out the result for each test case.

Reply:
Ah, I see! It's the same for `pytest`. Can you add verbosity to the `ctest` call in `test.yml`? Then I think we're good to go!